recipes-kernel/linux/linux-yocto-dev/qcm6490-board-dts/0001-arm64-dts-qcom-sc7280-don-t-enable-GPU-on-unsupporte.patch

```diff
@@ -1,7 +1,7 @@
-From 94d5ffab9d5ebc9caeab310f9d2eb36a65d7d3a9 Mon Sep 17 00:00:00 2001
+From 219114786221537b424886e18af5ea29b4ae41dc Mon Sep 17 00:00:00 2001
 From: Dmitry Baryshkov <dmitry.baryshkov@linaro.org>
 Date: Sat, 7 Sep 2024 15:51:25 +0300
-Subject: [PATCH] arm64: dts: qcom: sc7280: don't enable GPU on unsupported
+Subject: [PATCH 3/7] arm64: dts: qcom: sc7280: don't enable GPU on unsupported
```

Comment:
It says this patch is a backport from 6.13, do we really need to keep it?

Reply:
It was my mistake doing the rebase, I'm going to discard it